idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/gas_phase_thermo.py

```diff
@@ -312,7 +312,7 @@ def build(self):
         self.set_default_scaling("cp_mass", 1e-6)
         self.set_default_scaling("entr_mol", 1e-2)
         self.set_default_scaling("entr_mol_phase", 1e-2)
-        self.set_default_scaling("dens_mol", 1)
+        # self.set_default_scaling("dens_mol", 1)  # defined later
```

Comment:
Do we need this commented line?

Reply:
We don't need this since we define the scaling in the scaling factor method, I'll remove this line.

Reply:
@bpaul4 that line can be deleted.